frontend/src/components/shared/SearchHighlighter.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <span>
+    <span v-for="(char, j) in matchTerm.split('')" :key="j">
+      <mark v-if="isMatch(char)" class="has-background-warning">{{ char }}</mark>
```

Comment:
The `warning` is a bit too strong for me. I'm suggesting to use `bold` as in the Quick Search, and maybe also a pale grey. What do you think?

Come to think of it, it also looks like there is an extra `<span>` holding stuff together, perhaps that can be removed.

```suggestion
      <mark v-if="isMatch(char)" class="has-text-weight-bold has-background-grey-lighter">{{ char }}</mark>
```

Reply:
Thanks for the suggestion! I like the bold effect but don't like the grey background as much. How about having no background instead? Please see attached images for comparison. Note that we would probably switch from `mark` to `span` if we go with no background since `mark` is supposed to be for highlighted text.

We need the extra `span` as well since the inner `span` is with a `v-for` and Vue 2 does not allow multiple root elements.

<img width="920" alt="image" src="https://user-images.githubusercontent.com/423498/176417090-95380377-84a3-42de-b8b2-5fd55357f17d.png">

<img width="919" alt="image" src="https://user-images.githubusercontent.com/423498/176417306-fe236735-66cc-47bb-bacf-57a4c608c8ab.png">

Reply:
Nesting `template` doesn't work either since it gets removed after processing so it still ends up with multiple root elements.

Reply:
Style change is handled here: https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1051/commits/067c4b2f227c322ab8b7054da0666a7f59fec08e